docs/src/autodiff/overview.md

```diff
@@ -0,0 +1,22 @@
+# Using ChainRules in your AutoDiff system
+
+This section is for authors of AD systems.
+It assumes a pretty solid understanding of Julia, and of automatic differentiation.
+It explains how to make use of ChainRule's rule sets,
+to avoid having to code all your own AD primitives / custom sensitives.
+
+There are 3 main ways to access ChainRules rule sets in your AutoDiff system.
+
+1. [Operation Overloading Generation](operator_overloading.html)
+  - This is primarily intended for operator overloading based AD systems which will generate overloads for primal function based for their overloaded types based on the existance of an `rrule`/`frule`.
+  - A source code generation based AD can also use this by overloading their transform generating function directly so as not to recursively generate a transform but to just return the rule.
```

Comment:
This iliat sn't rendering correctly in the docs. I don't know why. Maybe try identing 4 spaces?